tdp/core/deployment/test_deployment_runner.py

```diff
@@ -135,10 +144,10 @@ def test_service_log_not_emitted_when_config_start_wrong_order(
     minimal_collections, deployment_runner
 ):
     """deployment plan containing start then config should not emit service log"""
-    deployment_plan = DeploymentPlan.from_operations(
-        minimal_collections, ["mock_node_start", "mock_node_config"]
+    deployment_log = DeploymentLog.from_operations(
+        minimal_collections, ["mock_start", "mock_config"]
```

Comment:
You do not need to change these operations.
```suggestion
        minimal_collections, ["mock_node_start", "mock_node_config"]
```